WcaOnRails/app/views/competitions/_admin_index_table.html.erb

```diff
@@ -33,13 +33,13 @@
           <tr>
             <td>
               <i class="flag f16 <%= competition.country.iso2.downcase %>"></i><%= link_to competition.cellName, competition_path(competition) %>
-              <br /><strong><%= competition.country.name_in(:en) %></strong>, <%= competition.cityName %>
```

Comment:
Whoops! This was so long ago, I don't know what I was thinking... if anything. Will revert my changes to this file.